package.json

```diff
@@ -133,12 +133,14 @@
     "@types/express": "4.17.3",
     "cors": "^2.8.5",
     "express": "^4.17.1",
+    "jws": "^3.0.0",
     "lodash": "^4.17.14"
   },
   "devDependencies": {
     "@types/chai": "^4.1.7",
     "@types/chai-as-promised": "^7.1.0",
     "@types/jsonwebtoken": "^8.3.2",
+    "@types/jws": "^3.2.4",
```

Comment:
I'm just curious, what's the process for adding a new package into the SDK?

Reply:
Great question - I haven't thought of it and I haven't been flagged by my lack of insight. I'll follow internal documentations to audit the dependencies I'm adding.